install/kubernetes/cilium/files/cilium-envoy/configmap/bootstrap-config.json

```diff
@@ -0,0 +1,307 @@
+{
+  "node": {
+    "id": "host~127.0.0.1~no-id~localdomain",
+    "cluster": "ingress-cluster"
+  },
+  "staticResources": {
+    "listeners": [
+      {{- if and .Values.proxy.prometheus.enabled .Values.envoy.prometheus.enabled }}
+      {
+        "name": "envoy-prometheus-metrics-listener",
+        "address": {
+          "socket_address": {
+            "address": "0.0.0.0",
```

Comment:
Does this port need to be open on all interfaces, or can it be limited to localhost? What about IPv6-only installations?

Reply:
@jrajahalme prometheus scrapes the prometheus data via the pods IP - after resolving all pod ips via the service (which needs to be headless for this purpose). therefore, i assume that this needs to be exposed on all interfaces.

otherwise - this will be addressed with https://github.com/cilium/cilium/issues/23353 anyway.